timing/timeFrobeniusFactor.cpp

```diff
@@ -66,15 +62,16 @@ int main(int argc, char* argv[]) {
 
   // Build graph
   NonlinearFactorGraph graph;
-  // graph.add(NonlinearEquality<SO4>(0, SO4()));
+  // graph.add(NonlinearEquality<SOn>(0, SOn::identity(4)));
```

Comment:
commented out code

Reply:
Yeah - not fixing that now. Was there before and orthogonal to PR